Doc/howto/enum.rst

```diff
@@ -459,6 +459,30 @@ sense to allow sharing some common behavior between a group of enumerations.
 (See `OrderedEnum`_ for an example.)
 
 
+.. _dataclass_support:
+
+Dataclass support
+-----------------
+
+When inheriting from a :func:`dataclass` the :func:`repr` omits the inherited
+class' name.  For example::
+
+    >>> @dataclass
+    ... class CreatureDataMixin:
+    ...     size: str
+    ...     legs: int
+    ...     tail: bool = field(repr=False, default=True)
+    ...
+    >>> class Creature(CreatureDataMixin, Enum):
+    ...     BEETLE = 'small', 6
+    ...     DOG = 'medium', 4
+    ...
+    >>> Creature.DOG
+    <Creature.DOG: size='medium', legs=4>
```

Comment:
```suggestion
    >>> repr(Creature.DOG)
    "<Creature.DOG: size='medium', legs=4>"
```

Maybe a little clearer to explicitly call `repr`, rather than just having the interactive interpreter implicitly print the repr()? Up to you, of course.

Reply:
I"m going to leave this one as-is as I think it matches the rest of the enum documentation better.